src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -104,6 +104,13 @@ private void openWindow(Stage mainStage) {
             }
         });
         Platform.runLater(this::openDatabases);
+
+        if(!(Globals.getFileUpdateMonitor().isActive())) {
+            this.getMainFrame().getDialogService()
+                    .showErrorDialogAndWait("Unable to monitor file changes. Please close files " +
```

Comment:
Please use `Localization.lang`. See https://devdocs.jabref.org/getting-into-the-code/code-howtos#using-localization-correctly.

Reply:
https://github.com/JabRef/user-documentation/pull/286